shell/deepdive-load

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# deepdive-load -- Loads a table from a file or Pipe's output.
+# You must either have the DEEPDIVE_DB_URL environment set to a proper URL or
+# run this command under a DeepDive application where the URL is set in a db.url file.
+#
+# > deepdive load table file format
+#
+# > deepdive load sentences sentences.tsv tsv
+# > deepdive load sentences sentences.csv csv
+#
+##
+set -eu
+
+# parse database settings
+. load-db-driver.sh
+
+if [[ $# -gt 1 ]]; then
+    table=$1; shift
+    file=$1; shift
+    if [[ $# -gt 0 ]]; then
+        format=$1; shift
+    else
+        format=tsv
+    fi
+    db-load $table $file $format
+else
+    error "Missing table name or file name"
```

Comment:
Please move this usage error checking to the top to avoid unnecessary global if blocks:
```bash
# ...
. load-db-driver.sh

[[ $# -gt 1 ]] || error "Missing table name or file name"

table=$1; shift
# ...
```